galaxy_ng/app/api/ui/viewsets/synclist.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+from galaxy_ng.app.api import base as api_base
+from galaxy_ng.app import models
+from galaxy_ng.app.api.ui import serializers
+
+
+log = logging.getLogger(__name__)
+
+
+class SyncListViewSet(api_base.ModelViewSet):
```

Comment:
I think we should make this endpoint admin only if we're going to keep it.

Reply:
Agree. Please limit this to an `admin` or `system:partner-engineer` user.